apps/rehabstod/src/hooks/useLogout.ts

```diff
@@ -1,8 +1,8 @@
 import { useNavigate } from 'react-router-dom'
-import { useFakeLogoutMutation, useGetUserQuery } from '../store/api'
+import { api, useFakeLogoutMutation } from '../store/api'
 
 export function useLogout() {
-  const { data: user } = useGetUserQuery()
+  const { data: user } = api.endpoints.getUser.useQueryState()
```

Comment:
This will load existing user data, but will never trigger a fetch. `useGetUserQuery` will return `{data: undefined}` when `session.authenticated` is false.